recipes-kernel/linux-yocto/linux-yocto/0001-iio-tmp006-Set-correct-iio-name.patch

```diff
@@ -0,0 +1,33 @@
+From a1dbe23dac5f38c92714f69d0cac749c5e954aa7 Mon Sep 17 00:00:00 2001
+From: Yong Li <yong.b.li@intel.com>
+Date: Fri, 22 Apr 2016 11:28:24 +0800
+Subject: [PATCH] iio: tmp006: Set correct iio name
+
+When load the driver using the below command:
+echo tmp006 0x40 > /sys/bus/i2c/devices/i2c-0/new_device
+
+In sysfs, the i2c name is tmp006, however the iio name is 0-0040,
+they are inconsistent. With this patch,
+the iio name will be the same as the i2c device name
```

Comment:
There is a thread is discussing this patch: http://www.spinics.net/lists/linux-iio/index.html#24331

In kernel, almost all I2C device drivers are using the id name as the IIO name, this name should be the same as the I2C device name.
Someone is discussing compatibility issues with some legacy applications. I do not think OStro has such kind of legacy compatibility issue. 

If there are more than one devices, fbp files can use the IIO device number for different devices, for example, iio:device0, iio:device1 ...